packages/block-editor/src/components/block-types-list/style.native.scss

```diff
@@ -0,0 +1,7 @@
+.rowSeparator {
+	height: 12px;
+}
+
+.columnPadding {
+	padding: $grid-unit-20;
+}
```

Comment:
Not to be picky but since we moved this, could we use the new naming? What do you think?

Reply:
Yeah, I totally agree, I used the BEM naming for the tabs component but I forgot to apply it here, thanks for pointing it out 🙇 !

Reply:
I changed the naming in this [commit](https://github.com/WordPress/gutenberg/pull/28495/commits/958dd9d19ebb35d73cd2d23d4211f98f569f6c67).